pkg/apiserver/apiserver.go

```diff
@@ -58,6 +58,7 @@ import (
 	"antrea.io/antrea/pkg/apiserver/registry/stats/antreanetworkpolicystats"
 	"antrea.io/antrea/pkg/apiserver/registry/stats/multicastgroup"
 	"antrea.io/antrea/pkg/apiserver/registry/stats/networkpolicystats"
+	nodelatencystat "antrea.io/antrea/pkg/apiserver/registry/stats/nodelatencystats"
```

Comment:
the alias doesn't seem necessary and makes it inconsistent with other similar xxxstats imports

Reply:
Ok, I have fixed it.